js/plugin/mediaPlayer.ts

```diff
@@ -79,7 +79,7 @@ export const MediaPlayerMixin = (SuperClass) => {
         this._player_update();
       });
 
-      this.addEventListener("browser-mod-connected", () =>
```

Comment:
`-connected` is never enough to tell we can actually communicate with component counter-part. Therefore a new event.

Eventually – we can rename `browser-mod-ready` to `browser-mod-connected` and keep just one event.